docker/docker-compose.yml

```diff
@@ -42,32 +31,40 @@ mesosslave1:
   - MESOS_CONTAINERIZERS=docker,mesos
   - DOCKER_DAEMON_ARGS="--log-level=error"
   links:
-  - etcd
   - mesosmaster1
-  - "ambassador:apiserver"
   volumes:
   - /var/run/docker.sock:/var/run/docker.sock
+  - /sys:/sys
+  - /usr/local/bin/docker:/usr/bin/docker
```

Comment:
Why is the host's docker binary and sys being mounted? Doesn't the container include docker?